sage.py

```diff
@@ -1238,11 +246,12 @@ def make_attack_graphs(victim_episodes, state_sequences, sev_sinks, datafile, di
 port_services = load_iana_mapping()
 
 print('------ Reading alerts ------')
-(team_alerts, team_labels) = load_data(path_to_json_files, alert_filtering_window)
-plot_histogram(team_alerts, team_labels)
+team_alerts, team_labels, team_start_times = load_data(path_to_json_files, alert_filtering_window, start_hour, end_hour)
+plot_histogram(team_alerts, team_labels, experiment_name)
+team_data = group_alerts_per_team(team_alerts, port_services)
```

Comment:
As mentioned above, `team_data` can directly be passed to the `aggregate_into_episodes` function and it does not have to deal with port-service mapping